lib/raxx/http1.ex

```diff
@@ -338,7 +338,7 @@ defmodule Raxx.HTTP1 do
           {:ok, Enum.reverse(headers), rest}
 
         {:ok, {:http_header, _, key, _, value}, rest} ->
-          parse_headers(rest, options, [{String.downcase("#{key}"), value} | headers])
+          parse_headers(rest, options, [{String.downcase("#{key}"), String.downcase("#{value}")} | headers])
```

Comment:
I think the value in http headers is case sensitive. only they key is case insensitive.
Started talking at you in the slack room but actually think now that might not find you

Reply:
Heya, yeah I wondered whether this might be hitting it with a sledge hammer - maybe the other PR is a better solution for this :+1:

Reply:
Agreed, well I'll give @CharlesOkwuagwu a few days to add tests and update changelog - otherwise I'll updated this branch with his solution.

Reply:
Want to tackle this. otherwise I'll creatively smoosh these two PR's later on.

Also to not running the formatter is part of CI on this project, so you'll need to do that if finishing off PR